testcase/discover.go

```diff
@@ -33,10 +33,12 @@ func discoverTestDirectory(path string) ([]TestCaseSet, error) {
 	}
 	var result []TestCaseSet
 	for _, f := range files {
-		if f.IsDir() || !strings.HasSuffix(f.Name(), ".json") {
+		log.Debugf("Read file: %s", f.Name())
+		if f.IsDir() || (!strings.HasSuffix(f.Name(), ".json") && !strings.HasSuffix(f.Name(), ".yaml")) {
```

Comment:
As noted in a review comment for another file I suggest we allow both .yaml and .yml.